apps/backend/src/modules/ai-assistant/libs/helpers/change-task/change-task-prompt-template.enum.ts

```diff
@@ -0,0 +1,41 @@
+const ChangeTaskPromptTemplates = {
```

Comment:
```suggestion
const ChangeTaskPromptTemplate = {
```
All enums should be singular